tests/Behat/AuthenticationContext.php

```diff
@@ -137,4 +137,41 @@ public function setNewUsersBlocked(): void {
     $this->configContext->setConfig('user.settings', 'register', USER_REGISTER_VISITORS_ADMINISTRATIVE_APPROVAL);
   }
 
+  /**
+   * We reset the authentication mock to the state as it was.
+   *
+   * @AfterScenario @ecas-login
+   */
+  public function resetAuthenticationMock(): void {
+    $this->visitPath('user/login');
+    if ($this->getSession()->getPage()->hasLink('Change it')) {
+      $this->getSession()->getPage()->clickLink('Change it');
+    }
+  }
+
+  /**
+   * Allow external users to login.
+   *
+   * @BeforeScenario @AllowExternalLogin
+   */
+  public function allowExternalUsers(): void {
+    // Set the assurance level to allow also external users to login. Security
+    // implications are mitigated by the fact that users are by default blocked
+    // when they login for the first time if they don't yet exist.
+    $config = \Drupal::configFactory()->getEditable('oe_authentication.settings');
+    $config->set('assurance_level', 'LOW');
+    $config->save();
```

Comment:
Can't we used the gathered ConfigContext and set this like that? This way we don't have to revert it ourselves in an after scenario?